src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -1558,12 +1546,13 @@ LedgerMaster::~LedgerMaster ()
 {
 }
 
-bool LedgerMaster::shouldAcquire (std::uint32_t currentLedger,
```

Comment:
LedgerIndex is better than std::uint32_t types for things specifically for ledger index values.